instance/models/instance.py

```diff
@@ -139,17 +143,37 @@ def active_server_set(self):
         return self.server_set.exclude_terminated()
 
     @property
-    def status(self):
+    def current_server(self):
         """
-        Instance status
+        Current active server. Raises InconsistentInstanceState if more than
+        one exists.
         """
         active_server_set = self.active_server_set
         if not active_server_set:
-            return self.EMPTY
+            return
```

Comment:
@chromano I think this makes sense.  You could consider prepending `current_server` with an underscore to mark it for internal use only.